sdks/java/core/src/test/java/org/apache/beam/sdk/coders/CommonCoderTest.java

```diff
@@ -63,6 +63,7 @@
       .put("urn:beam:coders:bytes:0.1", ByteCoder.class)
       .put("urn:beam:coders:kv:0.1", KvCoder.class)
       .put("urn:beam:coders:varint:0.1", VarLongCoder.class)
+      .put("urn:beam:coders:stream:0.1", IterableCoder.class)
```

Comment:
Suspect you need the `asCloudObject`-type stuff as in #1984. This will enable the Fn API to recognize the coder.